explore/src/main/scala/explore/targeteditor/AladinCell.scala

```diff
@@ -123,37 +121,39 @@ object AladinCell extends ModelOptics with AladinCommon:
   }
 
   private given Reusability[Props] =
-    Reusability.by(x => (x.uid, x.tid, x.obsConf, x.asterism, x.fullScreen.get))
+    Reusability.by(x =>
+      (x.uid,
+       x.tid,
+       x.obsConf,
+       x.asterism,
+       x.fullScreen.reuseByValue,
```

Comment:
`.get` has the same effect upon reusability here than `.reuseByValue` and avoids creating `Reuse` instances.